include/domain_bridge/topic_bridge_options.hpp

```diff
@@ -85,6 +86,16 @@ class TopicBridgeOptions
   TopicBridgeOptions &
   bidirectional(const bool & bidirectional);
 
+  /// Get reversed option.
+  DOMAIN_BRIDGE_PUBLIC
+  const bool &
+  reversed() const;
+
+  /// Set reversed option.
```

Comment:
It might be good to elaborate in the documentation here what happens when this is set to `true`.

Reply:
Documented this and the `bidirectional` option in 7b06684